prog/weave-npc/main.go

```diff
@@ -125,28 +125,19 @@ func createBaseRules(ipt *iptables.IPTables, ips ipset.Interface) error {
 		}
 	}
 
+	// If the destination address is not any of the local pods, let it through
 	if err := ipt.Append(npc.TableFilter, npc.MainChain,
-		"-m", "state", "--state", "NEW", "-j", string(npc.DefaultChain)); err != nil {
+		"-m", "physdev", "--physdev-out=vethwe-bridge", "-j", "ACCEPT"); err != nil {
```

Comment:
Hard-coding `vethwe-bridge` here seems wrong.  

Can this rule go in `weaver` where the rules go that drive traffic through the weave-npc chains?
Note that weave-npc is not otherwise tied to the specific implementation of Weave Net.

Reply:
removed hard-coded bridge name. PTAL

> Can this rule go in weaver where the rules go that drive traffic through the weave-npc chains?

No. moving it to `weaver` means this scenario will not work with kubenet etc. It fits in well in weave-npc as part of bunch of base rules that gets configured.